lib/account/account.controllers.js

```diff
@@ -17,29 +17,53 @@ module.exports = (models) => {
 
       return account;
     },
-    // getAccounts: async function(request, h) {
-    //   return models.Account.findAll()
-    //   .then(response => {
-    //     return response;
-    //   })
-    //   .catch(error => {
-    //     return error;
-    //   });
-    // },
+
+    getAccounts: async function(request, h) {
```

Comment:
If you could leave a TODO here to limit this route to superadmin that'd be good.